cli/gluster-block.c

```diff
@@ -1029,13 +1029,18 @@ main(int argc, char *argv[])
   int count = 1;
   int args = argc;
   int json = GB_JSON_NONE;
+  int ret;
 
 
   if (argc <= 1) {
     glusterBlockHelp();
     goto fail;
   }
 
+  if (initGbConfig()) {
```

Comment:
I don't see a reason why we need to make the gbConf go in shared memory for cli (gluster-block process)
Did you just called it coz you might want to make the lock initialization and gbConf allocation common ?
I don't see any critical issue by doing this, but just raising the hand.

Reply:
@pkalever 
Me too, it make no sense here :-), and will fix it.
Thanks.